lib/components/terms.js

```diff
@@ -144,11 +144,34 @@ export default class Terms extends React.Component {
             left: 0;
             bottom: 0;
             color: #fff;
-            transition: ${isMac ? 'none' : 'margin-top 0.3s ease'};
           }
 
           .terms_termsShifted {
             margin-top: 68px;
+            animation: shift-down 0.2s ease-out;
```

Comment:
I made the shift-down a little faster (and eased out) because it overlaps a bit with the tabs